src/extlibs/hrvo/path.h

```diff
@@ -0,0 +1,72 @@
+#pragma once
+#include <memory>
+#include <vector>
+
+#include "extlibs/hrvo/path_point.h"
+#include "extlibs/hrvo/vector2.h"
+
+class PathPoint;
+
+/**
+ * An agent's path
+ */
+class Path
+{
+   public:
+    /**
+     * Default constructor for this path
+     */
+    explicit Path();
+
+    /**
+     * Constructor for this path
+     * @param path_points A vector that stores path points, which can be empty
+     * @param goal_radius The goal radius for this path
```

Comment:
what does goal radius mean?

Reply:
`goal_radius` is suppose to be the max distance away from any path point. It's a way to verify if the agent is at the path point.

I can change the naming just to be consistent: `path_radius`